Beta tester-page code cleaned

LudvigStridsberg opened the pull request:
Removed unnecessary html-elements and unnecessary css-rules. Also made it look a bit closer to the provided pdf by, for example, making the video bigger.

### Screenshot
![Screenshot_2021-04-14 Snatch(1)](https://user-images.githubusercontent.com/77850704/114721315-1b33ac00-9d39-11eb-8bd8-c828b59d602f.png)

LudvigStridsberg committed:
Beta tester-page code cleaned

Removed unnecessary html-elements and unnecessary css-rules. Also made it look a bit closer to the provided pdf by, for example, making the video bigger.

VilmaNilsson reviewed (approved):
So much cleaner code than what I did, great!!

VilmaNilsson reviewed (changes requested):
There are some conflicts with this code, otherwise is it good! Much cleaner code than what I did

LudvigStridsberg commented:
I see that there are changes requested but I don't get what they are supposed to be. Mind explaining, please?

LudvigStridsberg committed:
Changes due to conflict

This will hopefully fix the conflict in the css. Also noticed that the margin dissapeared between the logo and the video so I added that as well.

Co-Authored-By: Vilse <77903212+VilmaNilsson@users.noreply.github.com>
